packages/vue/src/switch/SwitchThumb.vue

```diff
@@ -29,6 +29,6 @@ const { componentRef } = useComponentRef<HTMLInputElement | null>()
     v-bind="$attrs"
     ref="componentRef"
     :data-state="getState(inject.checked.value)"
-    :data-disabled="inject.disabled ? '' : undefined"
+    :data-disabled="inject.disabled?.value ? '' : undefined"
```

Comment:
Could you please validate this, and why it always appears without using `.value` inside the `<template>`

Reply:
yes, I don't know the technical reason behind it but .value sfc adjusts itself. So it doesn't used.

Reply:
inject with .value used. I think the data stays connected up there, otherwise it won't work.